applications: nrf5340_audio: Use file suffix for nrf5340_audio DFU (#15907)
nrfconnect/sdk-nrf: applications/nrf5340_audio/doc/building.rst

>    Alternatively, you can add the ``-d`` parameter to the ``west`` command to specify a custom build folder. This lets you build firmware for both

Reviewer: Should we remove the note on line 191-193 about the script being recommended?

Author: Discussed with the team and agreed it is time to remove this note

   headset and gateway before programming any development kits.

> Application Configuration Files

Reviewer 2: ```suggestion
Application configuration files
```

===============================

The application uses a :file:`prj.conf` configuration file located in the sample root directory for the default configuration.
It also provides additional files for different custom configurations.
When you build the sample, you can select one of these configurations using the :makevar:`FILE_SUFFIX` variable.

See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information.

The application supports the following configuration types:

> .. list-table:: Application Build Types

Reviewer 2: ```suggestion
.. list-table:: Application ´build types
```

   :widths: auto
   :header-rows: 1

   * - Build type
     - File name
     - Description
   * - Debug (default)
     - :file:`prj.conf`
     - Debug version of the application. Provides full logging capabilities and debug optimizations to ease development.
   * - Release
     - :file:`prj_release.conf`
     - Release version of the application. Disables logging capabilities and disables development features to create a smaller application binary.
   * - FOTA DFU
     - :file:`prj_fota.conf`

>      - | Builds the debug version of the application with the features needed to perform DFU over BLE, and includes bootloaders so that the applications on both the application core and network core can be updated.

Reviewer 2: ```suggestion
     - | Builds the debug version of the application with the features needed to perform DFU over Bluetooth LE, and includes bootloaders so that the applications on both the application core and network core can be updated.
```
